src/App.jsx

```diff
@@ -1,34 +1,24 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
 import "./App.css";
+import Header from "./components/header/Header";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getArticles } from "./utils/api";
 
 function App() {
-  const [count, setCount] = useState(0);
+  const [items, setItems] = useState([]);
+  const { items_types } = useParams();
+
+  useEffect(() => {
+    getArticles(items_types).then((articlesFromApi) => {
+      setItems(articlesFromApi);
+    });
+  }, [items_types]);
+  console.log("articleApi", items);
```

Comment:
You shouldn't need a useEffect in your App. The App should be rendering a component to display all the articles where this useEffect would then live.